event_registration_partner_unique/__manifest__.py

```diff
@@ -3,11 +3,12 @@
 # Copyright 2018 Tecnativa - Cristina Martin R.
 # Copyright 2020 Tecnativa - Víctor Martínez
 # Copyright 2023 Tecnativa - Carolina Fernandez
+# Copyright 2024 Tecnativa - Juan José Seguí Borja
```

Comment:
Normally for such small migration changes, better put yourself in the contributors.md file